yaksh/templates/yaksh/add_question.html

```diff
@@ -127,6 +127,22 @@ <h3>Uploaded files</h3>
         <h3>Test Cases</h3>
       </div>
       <div class="card-body">
+        {% if question %}
```

Comment:
There is a repeated if condition present in the question HTML table. Please remove that.